addons/hr/models/hr_employee.py

```diff
@@ -139,6 +139,10 @@ def _lang_get(self):
         help="Categorize your Employees by type. This field also has an impact on contracts. Only Employees, Students and Trainee will have contract history.")
 
     job_id = fields.Many2one(tracking=True, index=True)
+    activity_plans_ids = fields.Many2many('mail.activity.plan', string="Activity Plans",
+        groups="hr.group_hr_user", ondelete="cascade",
+        compute='_compute_activity_plans_ids', store=True)
```

Comment:
isn't this basically a related to `activity_ids.activity_plan_id`? If the field is related, I thing the groupby is possible without the need to store

Reply:
Would even say, who is going to group by activity plan ? Isn't using the activity view to check activities sufficient ? Seems like adding yet another m2m for niche use case to me.